flexprep/config/settings.yaml

```diff
@@ -10,11 +10,11 @@ main:
   db_path: /src/db/sqlite3-db
   s3_buckets:
     input:
-      endpoint_url: https://object-store.os-api.cci1.ecmwf.int
-      name: flexpart-input
+      endpoint_url: "${INPUT_FLEXPREP_S3_ENDPOINT_URL}"
```

Comment:
If possible, I would rename this variable to `FLEXPREP_INPUT_S3_ENDPOINT_URL`. Same for the other variables.